Northwind/Templates/EFCore/DbContext.t4

```diff
@@ -132,7 +115,7 @@ public partial class <#= Options.ContextName #> : DbContext
             var defaultValueSql = property.GetDefaultValueSql();
             if (defaultValueSql != null)
             {
-#>.HasDefaultValueSql("<#= defaultValueSql #>")<#
+#>.HasDefaultValueSql(<#= code.Literal(defaultValueSql) #>)<#
```

Comment:
This could be used for all literals, but it's really only needed when they might require escaping.